packages/jni-swig-stub/src/main/jni/realm_api_helpers.cpp

```diff
@@ -273,7 +273,7 @@ class CustomJVMScheduler {
     }
 
     void notify() {
-        auto jenv = get_env(true);
+        auto jenv = get_env(true, true, "core-notifier");
```

Comment:
@kiburtse If we don't know when the thread is done, then we cannot detach it from the JVM and this means that the thread will continue to exist until the process is terminated. I guess you can say that it doesn't take up that many resources with a daemon thread, but it will keep appearing in thread dumps, etc. It is quite disturbing when opening an closing many realms in our tests.